ui/src/components/table/table-element/grouping-path-cell.tsx

```diff
@@ -0,0 +1,56 @@
+import { ClipboardIcon } from 'lucide-react';
+import { Input } from '@/components/ui/input';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip';
+import { useState } from 'react';
+
+const GroupingPathCell = ({ data, uniqueId }: { data: string; uniqueId: string }) => {
```

Comment:
Could you rename the `data` prop to `path` so that it is more specific, thanks.

Reply:
done, thank you so much